browser-emulator-builder/lib/generatePolyfill.ts

```diff
@@ -172,8 +172,12 @@ function get<T>(obj: T, path: string): T {
   while (split.length) {
     const key = split.shift();
     const next = current[key];
-    if (next) current = next;
-    else if (split.length && split[0] === 'prototype') {
+
+    if (next) {
+      current = next;
+    } else if (key === '_$otherInvocations') {
+      return current[`${key}.${split.join('.')}`];
```

Comment:
Is this one condition supposed to be returning?

Reply:
Yea it always goes down in the object and path is split by '.' but the `otherInvocations` contain '.' in the key so it thinks its a nested path while we are already at the deepest level